### PR TITLE
Initial Website

### PR DESCRIPTION
**Pull request by 1337kiwi**
Hi all. First Commit. Revamped the website, added in goodies, and so on. 
Moving TItle is up for debate. I personally like it but I think we should vote on it, 
Support for most browsers (list in comments), links to other resources work, and so on. Complete change of the website structure, linking to the demo site added.

Regarding Text/Colours: I have synesthesia which means I perceive colours in super weird ways and identify colours differently (7 is yellow gang WYA). I also have no clue how to write things in advertiser-ey ways. I've used colours to match the branding colour scheme. Please give as many ideas or recommendations for colour scheme and/or text changes as you can.  Credit to @Bankst for the favicon.

### CODE REVIEW

**Comment by 1337kiwi** [comment]
Tested on Linux Firefox and Chromium. Used User-Agent Switcher to test on these platforms:

Mac OS X / Safari 12

Windows / IE 11
Windows / Edge 44
Windows / Chrome 83
Windows / Firefox 78
Windows / Firefox 60 ESR

Android Phone / Firefox 78
Android Phone / Chrome 83
Android Tablet / Firefox 78
Android Tablet / Chrome 83

iPhone / Safari 12.1.1
iPad / Safari 12.1.1

Google Bot
PS4
Curl
Wget

**Comment by neel-j-dev** [comment]
[Project landing page (like Chameleon has)](https://trello.com/c/9KP0vgU6/25-project-landing-page-like-chameleon-has)

File: index.html [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <html> <!-- TODO: We need more images and videos to improve quality of the site, as well as other design stuff-->
     <head>
-        <title>P</title>
+        <title>Photon</title>
```
Comment (prateekma) [inline index.html]: I think the title should be "Photon Vision" instead of just "Photon".

**Review by pietroglyph — APPROVED** [review]
Looks good enough to merge.

**Pull request merged**